modules/@ergonode/core/src/mixins/modals/actionModalFormMixin.js

```diff
@@ -29,8 +29,10 @@ export default function ({
                 this.removeErrors();
 
                 request().then(response => response.default({
```

Comment:
actionModalFormMixin should not be used - it's been replaced by new mechanism of autosave